CODEOWNERS

```diff
@@ -1 +1 @@
- * @crfroehlich @davehensleyns8 @CTDoriott @andrewnoyes @rob-ns8
+  * @crfroehlich @davehensleyns8 @CTDoriott @rob-ns8 @aldahick
```

Comment:
Looks like an extra space snuck in at the beginning. Probably doesn't matter but ... 🤷‍♂️

Reply:
Yeah, this seems to have happened [here](https://github.com/ns8inc/protect-sdk-php/pull/88/files) too.